artifacthub-pkg.yml

```diff
@@ -0,0 +1,28 @@
+---
+version: 0.1.0
+name: env-variable-secrets-scanner-policy
+displayName: Environment Variable Secrets Scanner
+createdAt: '2022-07-19T16:39:10+02:00'
+description: Reject Pods that contain secrets in an environment variable
+license: Apache-2.0
+homeURL: https://github.com/kubewarden/env-variable-secrets-scanner-policy
+containersImages:
+- name: policy
+  image: "ghcr.io/kubewarden/policies/env-variable-secrets-scanner-policy:v0.1.0"
+keywords:
+- secrets
+- api keys
+- tokens
+links:
+- name: policy
+  url: https://github.com/kubewarden/env-variable-secrets-scanner-policy/releases/download/v0.1.0/policy.wasm
+- name: source
+  url: https://github.com/kubewarden/env-variable-secrets-scanner-policy
+provider:
+  name: kubewarden
+recommendations:
+- url: https://artifacthub.io/packages/helm/kubewarden/kubewarden-controller
+annotations:
+  kubewarden/resources: Pod # comma separated list
```

Comment:
You have to add all the other higher level resources that can be targeted

Reply:
done https://github.com/kubewarden/env-variable-secrets-scanner-policy/pull/1/commits/8f58c5590c94d0d236a348a706ea4d9d412981ca